flake.nix

```diff
@@ -26,12 +13,14 @@
         repl = ./repl.nix;
         systemFlake = import ./systemFlake.nix { flake-utils-plus = self; };
 
-        modulesFromList = paths: genAttrs'
-          (path: {
-            name = removeSuffix ".nix" (baseNameOf path);
-            value = import path;
-          })
-          paths;
+        builders = {
+          overlaysPackagesBuilder = import ./overlaysPackagesBuilder.nix { flake-utils-plus = self; };
```

Comment:
`packagesFromOverlaysBuilderConstructor`

`Constructor` as it "constructs" a "builder" by injecting the `channels` dependency.

Reply:
For any name you do choose, I think `packagesBuilder` should be inside it just to be clear where this function is used and what its for.

Reply:
I think the name is actually good, maybe just reorder fromOverlaysPackagesBuilder

Reply:
It's analogous to the modulesFromListExporter / overlaysFromChannelExporter.

`modulesFromList` set the precedence here.